### DISCUSSION TITLE
自己權益受損才出來叫

### DISCUSSION THREAD

**Discussion by Tentiauhong** [WeNeedHome/SummaryOfLoanSuspension#954] [General]
你們中國人人自私啊，沒救了。
我支持河南警察，你們可以打我了。

**Comment by DancerCoding** [comment]
中国人生性擅长内斗，对内阴险狡诈，对外懦弱无能，在极权独裁者的压迫和颅内监禁下，人民被大量洗脑爱国（实质要爱党），初期面对自由主义世界的思想冲击，大部分本能地会维护极权的统治并为极权说话（粉红），但不代表所有大陆人不会抗争，至少目前来看，还是有希望。希望港澳台同胞一定坚守民主自由阵线，誓死捍卫最後留存的民主净土，誓死抗争拒绝被独裁政权吞并，否则中华民族将会在铁骑统治下下永无翻身之日。

**Reply by wedxz** [reply]
香港已经失守了

**Reply by markKscode** [reply]
> 香港已经失守了

香港和澳门没什么办法。说到底还是得靠台湾，不过我觉得中共大概率没法侵占台湾，除非就是把台湾给炸平了。相信到时候全球都会支援台湾。

**Reply by jianhao** [reply]
搁这鬼叫的，应该是湾湾的渣渣吧，我们是要过幸福的生活，但不是反对国家反对政党，我们是希望我们的国家发展的更好更强大繁荣富强民主，在这带nm节奏呢？？？

**Reply by Kasianian** [reply]
> 搁这鬼叫的，应该是湾湾的渣渣吧，我们是要过幸福的生活，但不是反对国家反对政党，我们是希望我们的国家发展的更好更强大繁荣富强民主，在这带nm节奏呢？？？

被独裁弑人政权圈养的牲畜，连自主生育权都没有的家猪，觉得生活在猪圈里才是正常的，你想过猪会绝育抗争吗？

**Reply by maxwillma** [reply]
南望王师每一天啊， 快点统一吧！

**Comment by Mmmmeruem** [comment]
我不是河南储户也不是烂尾楼业主，是不是该说我狗拿耗子了？

**Comment by markKscode** [comment]
没办法，中国就是这样，舆论被控制，为大众维权的正义人士全部都被扣帽子，坐大牢去了。那些事情大伙儿也都看不到，也无法闹起风浪。

**Comment by richardo876** [comment]
我认为应该减少讲这种“没救了”等的晦气话，几十年来的洗脑教育不是一朝一夕就能改变。 现在最低成本和安全的做法就是在网上键政揭露中共的阴暗（在内网可用所谓阴阳怪气的语言绕过审核，在外网就自由发言），这几年来，无论在知乎还是微博，肉眼可见的反体制的声音越来越多，还是能看到一点希望的。包子逆行倒施，终会尝恶果。

**Reply by maxwillma** [reply]
附议

**Comment by ZVMAGL3** [comment]
我可不是烂尾楼的业主，到那时我希望中国变好

**Reply by ZVMAGL3** [reply]
？

**Reply by ZVMAGL3** [reply]
> 

？

**Comment by Ventery** [comment]
1450台蛙子省省吧，下次起码交几次代码再来带节奏。一个个注册没几天的号给这带你妈节奏呢？

**Reply by hrsgn** [reply]
996结束了还是熬过35危机了？不去提交你的代码来这里干啥

**Reply by itsoursduty** [reply]
带节奏？我寻思人说的不是事实么

**Reply by tanuk1997** [reply]
你给你姥爷爹洗地会不会给你几份狗粮？

**Comment by k624035508** [comment]
我是鲁迅笔下吃人血馒头的人   看到别人的心血全摆烂在烂尾楼就想点赞   这就是小粉红的下场

**Comment by ytx0574** [comment]
呆哇?  遇到中国人, 中国人几乎都会提收复台湾, 武统台湾, 你弯弯人还不是学会了见人说人话, 见鬼说鬼话, 有啥好得瑟的. 一面赚咱这些大陆人的钱, 一面背后酸我们, 不是一样的得行?  奉承我们支撑收复, 其实心里跟吃了屎一样难受. 😄.    其实我们对收不收回你这小岛, 咱一点兴趣咩有, 收回来还不是吸我们的血.

**Comment by Dreaming-Pepper** [comment]
以前也觉得，事不关己高高挂起，去关心这事儿，是狗拿耗子。
但显然，是没意识到，自己和那些人属于同一利益共同体，非要屎到临头来搅便，最后被乱棍打醒，自己和他们是同意利益共同体，如果是就应该支持他们去斗争

**Comment by hqqlucky** [comment]
其实这次烂尾也好，经济不景气也好，直接武统呆湾倒是能续命10年，至于所谓的全球支援呆湾，乌克兰作为独立国家都没有得到这个待遇，你一个没战略纵深，依靠米帝的，甚至连国际上很多国家都不承认的地区，靠什么来拖时间打？乌克兰至少有战略纵深。你们高层也清楚，呆湾只要在打起来政府重要人员能第一时间跑到米帝就是伟大胜利了，所以经常演习怎么跑路。估计真打起来的话，最大的可能就是远点的国家意思一下，特别是欧洲那些国家对乌克兰已经焦头烂额了，哪有心情管我们这边，真正能出手的估计也就是日本会派点船，韩国自顾不暇，北方有朝鲜。然后澳大利亚会派一点船，至于米帝，一个两个航母战斗群不顶用，多了肯定也不敢，最主要的是我们武统绝对是闪电战的形式，让米帝根本反应不过来。大陆觉得最麻烦的就是武统之后怎么善后的问题，憨批多的是，主要是会煽动成巷战。

**Reply by hqqlucky** [reply]
中国确实有不好的地方，但是谁也别笑话谁，我润也不会润呆湾。更烂

**Reply by itsoursduty** [reply]
讲这么多有什么用 能掩盖问题吗？先不说能不能续命 续命十年然后呢？

**Reply by markKscode** [reply]
别拿着中国最发达的地方和台湾比，你为中国那悬殊的贫富差距、不均衡发展自豪呗，富人在中国的生活确实比台湾好，但我相信台湾普通百姓的生活绝对比我们的普通百姓好上很多很多。还有乌克兰这么一个小国，俄罗斯打成这样，那么攻打台湾会如何谁都不知道，还有西方各国这次团结一致对俄进行经济制裁，我相信打台湾，西方世界的经济制裁让普通百姓的日子难过好几倍。所以，我是绝对不支持攻打台湾的。

**Reply by lance111l** [reply]
台湾若想独立 肯定被痛打     台湾民众哭吧

**Comment by yang131323** [comment]
> 你們中國人人自私啊，沒救了。 我支持河南警察，你們可以打我了。

说你妹呢，人都是自私的，你摸着你良心问问你做到大公无私了吗？是个人就有私心
还你们中国人？那请问你是西方国家的中国🐶？
不盼着自己国家好，天天在这里阴阳怪气的，舔西方国家舔上瘾了是吧？每个国家都有做的不好的地方，不为国家越来越好出一份力，键盘倒是敲的响？说白点你就是想白嫖，看到好的东西你想直接享受，不想为这个东西做出贡献，你要是觉得国家哪点有不足，你用你自己的行动去影响和改善啊，除了口嗨你会干嘛？

**Reply by Tentiauhong** [reply]
不愧人礦，你們中國你們中國你們中國，太重要了再說三次。哈哈哈。
氣死吧，9元零時工一堆，寫個屁程式要賺幾毛啊。

**Comment by Tentiauhong** [comment]
上面幾個要打台灣的還活著嗎？還是跳樓了

**Comment by Cegazhos** [comment]
你们这些台湾傻狗和大陆脑残 全都死光最好了。一群废物

**Reply by maxwillma** [reply]
本区被封了？

**Reply by Cegazhos** [reply]
> 本区被封了？

石砸狗叫的台灣鄉巴佬 你好哇

**Reply by Tentiauhong** [reply]
維權維到剩你們幾隻粉紅戰狼了，你的房價剩幾成說來聽聽啊，還有響應下鄉了沒？

**Reply by Cegazhos** [reply]
> 維權維到剩你們幾隻粉紅戰狼了，你的房價剩幾成說來聽聽啊，還有響應下鄉了沒？

傻卵台湾狗藏不住了吧，吃你妈拉的鸡蛋去吧。

**Reply by Tentiauhong** [reply]
最新消息中國人偷渡到越南打工，你國真厲害啊。偷渡美國就算了，連東南亞都不放過。看習近平把你窮到什麼鳥樣了。一人做事一人當，把你座標說出來，我要保證臺灣導彈不會誤傷無辜。這麼無腦嗆人，該不會是腦殘整天在Github複製貼上的搬運工吧。

**Reply by Cegazhos** [reply]
> 最新消息中國人偷渡到越南打工，你國真厲害啊。偷渡美國就算了，連東南亞都不放過。看習近平把你窮到什麼鳥樣了。一人做事一人當，把你座標說出來，我要保證臺灣導彈不會誤傷無辜。這麼無腦嗆人，該不會是腦殘整天在Github複製貼上的搬運工吧。

第一 恁娘的馬眼居然比恁的心眼還小。兩周前被恁爹操了兩周後還來狗叫，恁娘高潮神經反應這麽慢?
第二 傻逼台灣狗 沒有恁米國爹的大吊 恁倒是按按鈕啊，拉屎還要恁米國爹同意的傻狗 恁倒是射啊 俺在北京 速度射 不射就是狗雜種 想要操恁娘的人一大堆 直接射 恁爹准了。
第三 恁的什麽幾把新聞跟恁爹有幾把關係啊 恁爹前面表態很明顯了 恁是眼睛被米國爹的陰毛擋住了嗎 還在那扯那個小學生，跟個傻鳥一樣 扣帽子都扣不明白 。
第四 恁爹在哪拉屎跟恁有啥關係 恁要來吃?扯不過就要扯現實了 恁可真是夠有腦的呀 腦殘

**Comment by b4o** [comment]
要战就去磨刀，评论区的台独港独们，美国控制了全球90%媒体给你们洗脑，但是这个世界就是要靠实力的，也是你们喜欢的方式，所以天天在网上叫嚣没什么用的，革命从来都是要流血的，你们怎么那么天真呢，真是奇怪，想要独立又不去当兵，哎，太可怜了
